src/kubeseal_auto/kubeseal.py

```
@@ -75,188 +134,277 @@ def create_generic_secret(self, secret_params: dict):
 
-        command = (
-            f"kubectl create secret generic {secret_params['name']} {secret_entries} "
-            f"--namespace {secret_params['namespace']} --dry-run=client -o yaml "
-            f"> {self.temp_file.name}"
-        )
-        ic(command)
+        ic(cmd)
 
-        subprocess.call(command, shell=True)
+        with open(self.temp_file.name, "w") as f:
+            subprocess.run(cmd, stdout=f, check=True)
```

Ruff (0.14.7)

`subprocess` call: check for execution of untrusted input

(S603)